src/plugins/foxPage/components/FoxTab.tsx

```diff
@@ -18,11 +17,11 @@ export const FoxTab: React.FC<FoxTabProps> = ({
   assetSymbol,
   fiatAmount,
   cryptoAmount,
-  isSelected,
   onClick,
   ...props
 }) => {
   const bgHover = useColorModeValue('gray.100', 'gray.750')
+  const { isSelected, ...otherProps } = props
```

Comment:
This doesn't produce warnings anymore, but not for the reason one would initially think. The reason it works is because we're not destructuring `isSelected` from the functional component arguments at runtime, but from the rest `...props`.
Using the [rest parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters) syntax effectively creates a new reference, so all react sanity checks are effectively gone here.

Reply:
Agreed !